recipe/meta.yaml

```
@@ -255,21 +267,22 @@ outputs:
         - setuptools
         - pip
         - numpy  {{ numpy }}
-        # OpenBLAS + libgomp for osx-arm64, and linux-s390x and linux-ppc64le, and linux-aarch64. 
-        - openblas 0.3.21         # [s390x or ppc64le or aarch64 or arm64]     
+        # OpenBLAS + libgomp for osx-arm64, and linux-s390x and linux-ppc64le, and linux-aarch64.
+        - openblas 0.3.21         # [s390x or ppc64le or aarch64 or arm64]
         # MKL for win, linux-64, and osx-64. All require intel-openmp.
-        - mkl 2023.1.0            # [not (s390x or ppc64le or aarch64 or arm64)] 
+        - mkl {{ mkl }}           # [not (s390x or ppc64le or aarch64 or arm64)]
         # mkl-devel seems to be required for win.
-        - mkl-devel 2023.1.0      # [win]
+        - mkl-devel {{ mkl }}     # [win]
         - libfaiss ={{ version }}=*_{{ faiss_proc_type }}
         - libfaiss-avx2 ={{ version }}=*_{{ faiss_proc_type }}  # [x86_64]
       run:
         - python
         - libfaiss ={{ version }}=*_{{ faiss_proc_type }}
         - libfaiss-avx2 ={{ version }}=*_{{ faiss_proc_type }}  # [x86_64]
         - {{ pin_compatible('numpy') }}
```

```suggestion
        - numpy
```
Bounds come from run_exports, and yield a higher range than pin_compatible in this case.